pixi.toml

```diff
@@ -70,3 +32,32 @@ requests = "*"
 scipy = "*"
 shapely = "*"
 xmipy = "*"
+
+[pypi-dependencies]
+flopy = { git = "https://github.com/modflowpy/flopy.git" }
+modflowapi = { git = "https://github.com/MODFLOW-USGS/modflowapi.git" }
+modflow-devtools = { git = "https://github.com/MODFLOW-USGS/modflow-devtools.git" }
```

Comment:
I already had that change locally.
There are two reasons why I didn't submit it in the end:
- This is a change in behavior, the git commit will be pinned in the lock file
- Our use of flopy is a bit special. We modify it with `update-flopy`. Let's keep that in mind in case problems with flopy appear.

Reply:
> This is a change in behavior, the git commit will be pinned in the lock file

good point, I hadn't thought this through all the way &mdash; anyone developing flopy alongside mf6 will need to check in an updated pixi lock file to pull in any changes to flopy that are not generated components

Reply:
Seems to me like we are going to want to develop using the most up to date version of flopy.

Reply:
Yes- I think there is still a case to keep the version locked if we want to keep going in the direction of deterministic CI? We can still develop against the latest flopy, it would just be explicit in mf6 version history when a change is made w.r.t. flopy compatibility. could make debugging easier at the cost of a busier commit log.